cli/pkg/kctrl/cmd/package/repository/list.go

```diff
@@ -113,7 +113,8 @@ func NewSourceValue(pkgr v1alpha1.PackageRepository) uitable.Value {
 		switch {
 		case pkgr.Spec.Fetch.ImgpkgBundle != nil:
 			source = "(imgpkg) " + pkgr.Spec.Fetch.ImgpkgBundle.Image
-			if pkgr.Spec.Fetch.ImgpkgBundle.TagSelection != nil && pkgr.Spec.Fetch.ImgpkgBundle.TagSelection.Semver != nil {
+			if pkgr.Spec.Fetch.ImgpkgBundle.TagSelection != nil && pkgr.Spec.Fetch.ImgpkgBundle.TagSelection.Semver != nil &&
+				pkgr.Spec.Fetch.ImgpkgBundle.TagSelection.Semver.Constraints != "" {
```

Comment:
Does this make sense or should we have something like v0.0.0+ if constraint is empty?

Reply:
package repository logic in kapp-controller is responsible for picking version if it's not provided by a human.